osu.Game.Tests/Visual/Multiplayer/TestSceneTeamVersus.cs

```diff
@@ -162,13 +162,13 @@ public void TestChangeTypeViaMatchSettings()
                 }
             });
 
-            AddAssert("room type is head to head", () => client.Room?.Settings.MatchType == MatchType.HeadToHead);
+            AddUntilStep("room type is head to head", () => client.Room?.Settings.MatchType == MatchType.HeadToHead);
```

Comment:
Same with this one, not required.

Reply:
I just changed all of these asserts across the entire dolls to until steps, because it was bothering me that it was half-and-half even before I came in.